package.json

```diff
@@ -9,8 +9,8 @@
     "preview": "vite preview"
   },
   "devDependencies": {
-    "vite": "^4.5.2",
-    "glob": "^8.1.0"
+    "glob": "^8.1.0",
```

Comment:
=) 

In global projects for the future. Please don't push any changes in package.json and package-lock.json if it's not decided.

Reply:
ok